imports/plugins/core/graphql/lib/hocs/withPrimaryShopId.test.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import { compose } from "recompose";
+import { mount } from "enzyme";
+import { MockedProvider } from "react-apollo/test-utils";
+import getPrimaryShopId from "../queries/getPrimaryShopId";
+import withPrimaryShopId from "./withPrimaryShopId";
+
+const fakeOpaqueShopId = "cmVhY3Rpb24vc2hvcDpKOEJocTN1VHRkZ3daeDNyeg==";
+const MockComponent = () => <div>Mock</div>;
+const TestComponent = compose(withPrimaryShopId)(MockComponent);
+
+test("renders child component with correct shop id", async () => {
```

Comment:
@aldeed Can you take a look at my first unit test for these HOC components? I want to make sure I have good patterns here before I write tests for the others.

Reply:
@dancastellon See a couple of minor comments on the diff. Generally seems like a good pattern though